bindings/julia/Displaz.jl

```diff
@@ -50,15 +49,54 @@ function write_ply_points(fileName, nvertices, fields)
             for i = 1:size(value,2)
                 propName = ply_property_name(semantic, i)
                 write(fid, "property $typeName $propName\n")
-            end
+        	end
         end
         write(fid, "end_header\n")
         for (_,value) in convertedFields
-            write(fid, value')
+			write(fid, value')
         end
     end
 end
 
+function write_ply_lines(fileName, position, color)
+	vertexValid = ~all(isnan(position),2)
+	lineStarts = find(x -> x == 1, diff([false;vertexValid],1))
+	nvalidvertices = sum(x -> x == true, vertexValid)
+
+	# Create and write to ply file
+	fid = open(fileName, "w")
+	write(fid, "ply\n")
+	write(fid, "format binary_little_endian 1.0\n")
+	write(fid,"element vertex $nvalidvertices\n")
+	write(fid,"property double x\n")
+	write(fid,"property double y\n")
+	write(fid,"property double z\n")
+	write(fid,"element color $nvalidvertices\n")
+	write(fid,"property float r\n")
+	write(fid,"property float g\n")
+	write(fid,"property float b\n")
+	write(fid,"element edge $(length(lineStarts))\n")
+	write(fid,"property list int int vertex_index\n")
+	write(fid,"end_header\n")
+	idx = find(x-> x== true,vertexValid)	
+	write(fid,position[idx,:]')
+	write(fid,color[idx,:]')
+
+	# Write out line connectivity
+    nvertices = size(position,1)
+    realStart = 0
+    for i = lineStarts'
+        j = i
+        while j <= nvertices && vertexValid[j]
+            j = j + 1
+        end
+        lineLen = j-i
+		write(fid,int32(lineLen))
+        write(fid,int32(realStart:realStart+lineLen-1))
```

Comment:
`int32` is deprecated in 0.4, it's now `Int32`.  You can also use `collect()` to create an array from any iterable such as the range `realStart:realStart+lineLen-1`:

```julia
write(fid, Int32(lineLen))
write(fid, collect(Int32, realStart:realStart+lineLen-1))
```